user/python-desktop-notifier/template.py

```diff
@@ -0,0 +1,24 @@
+pkgname = "python-desktop-notifier"
+pkgver = "6.0.0"
+pkgrel = 0
+build_style = "python_pep517"
+make_check_wrapper = ["dbus-run-session"]
+hostmakedepends = [
+    "python-build",
+    "python-installer",
+    "python-setuptools",
+]
+depends = ["python", "python-bidict", "python-dbus-fast", "python-packaging"]
```

Comment:
```suggestion
depends = ["python-bidict", "python-dbus-fast", "python-packaging"]
```

redundant, gets pulled in by the other stuff